avm/ptn/sa/chat-with-your-data/modules/app/adminweb.bicep

```diff
@@ -0,0 +1,139 @@
+param kind string = 'app,linux'
+
+@description('Optional. The managed identity definition for this resource.')
+param userAssignedIdentityResourceId string = ''
+
+@description('Optional. Diagnostic settings for the resource.')
+param diagnosticSettings array = []
+
+@description('Optional. To enable pulling image over Virtual Network.')
+param vnetImagePullEnabled bool = false
+
+@description('Optional. Virtual Network Route All enabled.')
+param vnetRouteAllEnabled bool = false
+
+@description('Optional. Azure Resource Manager ID of the Virtual network subnet.')
+param virtualNetworkSubnetId string = ''
+
+@description('Optional. Whether or not public network access is allowed for this resource.')
+param publicNetworkAccess string?
+
+@description('Optional. Configuration details for private endpoints.')
+param privateEndpoints array = []
+
+
+// Calculate the linuxFxVersion based on runtime or docker settings
+var linuxFxVersion = useDocker
+  ? 'DOCKER|${dockerFullImageName}'
+  : (empty(runtimeVersion) ? toUpper(runtimeName) : '${toUpper(runtimeName)}|${runtimeVersion}')
+
+// Site configuration
+var siteConfig = {
+  linuxFxVersion: linuxFxVersion
+  appCommandLine: useDocker ? '' : appCommandLine
+  healthCheckPath: healthCheckPath
+  cors: {
+    allowedOrigins: allowedOrigins
+  }
+  minTlsVersion: '1.2'
+}
+
+// Build the configs array expected by the child module (appsettings config)
+var appConfigs = [
+  {
+    name: 'appsettings'
+    properties: union(
+      appSettings,
+      {
+        SCM_DO_BUILD_DURING_DEPLOYMENT: string(scmDoBuildDuringDeployment)
+        ENABLE_ORYX_BUILD: string(enableOryxBuild)
+        AZURE_RESOURCE_GROUP: resourceGroup().name
+        AZURE_SUBSCRIPTION_ID: subscription().subscriptionId
+      },
+      runtimeName == 'python' && appCommandLine == '' ? { PYTHON_ENABLE_GUNICORN_MULTIWORKERS: 'true' } : {}
+    )
+    applicationInsightResourceId: empty(applicationInsightsName)
+      ? null
+      : resourceId('Microsoft.Insights/components', applicationInsightsName)
+    storageAccountResourceId: null
+    storageAccountUseIdentityAuthentication: null
+    retainCurrentAppSettings: true
+  }
+]
+
+module adminweb '../core/host/appservice.bicep' = {
+  name: '${name}-app-module'
+  params: {
+    name: name
+    location: location
+    tags: tags
+    allTags: allTags
+    kind: kind
+    serverFarmResourceId: serverFarmResourceId
+    siteConfig: siteConfig
+    configs: appConfigs
+    diagnosticSettings: diagnosticSettings
+    vnetImagePullEnabled: vnetImagePullEnabled
+    vnetRouteAllEnabled: vnetRouteAllEnabled
+    virtualNetworkSubnetId: virtualNetworkSubnetId
+    publicNetworkAccess: empty(publicNetworkAccess) ? null : publicNetworkAccess
+    privateEndpoints: privateEndpoints
+    managedIdentities: {
+      systemAssigned: false
+      userAssignedResourceIds: [
+        userAssignedIdentityResourceId
+      ]
+    }
+  }
+}
+
+output WEBSITE_ADMIN_NAME string = adminweb.outputs.name
```

Comment:
Also for all outputs using description metadata and one empty line as separator would be recommended